.github/workflows/on-pr_pr-labeler.yml

```diff
@@ -0,0 +1,17 @@
+name: "[on PR] PR Labeler"
+on:
+  pull_request_target:
+    types: [opened, reopened]
+
+jobs:
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: TimonVS/pr-labeler-action@v3
```

Comment:
whats the additional labeler for ?

Reply:
The labeler Danny recently brought removed the old labeler in app.
Now we have two, this one (the original before Danny's change) labels a PR based upon some criteria. Very useful as that label then feeds into the release drafter.
The new one adds our `review-requested` tag